Source/SelfService/Web/logging/logFilter/logFilterToLabelsAndPipeline.ts

```diff
@@ -22,7 +22,7 @@ export const logFilterToLabelsAndPipeline = (applicationId: string, environment:
                 : undefined,
     };
 
-    const pipeline = filters.searchTerms.map((term) => `|= "${term}"`);
+    const pipeline = filters.searchTerms.map((term) => `|~"(?i)${term}"`);
```

Comment:
```suggestion
    const pipeline = filters.searchTerms
        .map((term) => term.replace(/[.*+?^${}()|[\]\\"]/g, '\\$&'))
        .map((term) => `|~"(?i)${term}"`);
```

Reply:
Aha, so we're basicaly sending RegExp to loki to be parsed there. So with this escaping it would abstract that away from the user using the search field, so they could only search for plain text?

Reply:
After testing this out a bit locally, I ended up having to double-escape the special characters.

Reply:
This also matches what is sent when testing this regexp in grafana: `|~"(?i)\\?"`. Generates the same request to the server :)